Fix export failing on collections with virtual fields like $thumbnail by excluding them from export defaults and the field picker (#27073)
directus/directus: .changeset/curvy-tigers-exist.md

---
'@directus/app': patch
---

> Excluded virtual fields like `$thumbnail` from export defaults and the Add field picker in the export sidebar so JSON exports no longer failed from synthetic field selection.

Reviewer: ```suggestion
Fixed export failing on collections with virtual fields like $thumbnail by excluding them from export defaults and the field picker
```

Reviewer: Hi! Just a small thought on the changeset ! it could maybe be tightened up a bit

Author: Yeah makes sense, tightened it up. I'll commit the suggestion.
